### ISSUE TITLE
Wiper / snapshots get BSC archive node out of sync

### ISSUE THREAD

**Issue by koen84** [binance-chain/bsc#144]
#### System information

Geth version: `1.0.7`
OS & Version: Ubuntu 18.04 LTS

This is a dedicated server : AMD EPYC 7502P 32-Core Processor, 128 GB RAM and mirror NVMe drives

#### Expected behaviour
BSC node prioritises keeping synced to chainhead

#### Actual behaviour
After a while, with wiper and snapshot aborting / resuming, it starts lagging, by a lot even :
`t=2021-04-13T00:00:08+0200 lvl=info msg="Imported new chain segment"            blocks=8  txs=1405 mgas=152.871 elapsed=8.669s      mgasps=17.633 number=6509190 hash=0xb6f1e8d7a52198f653f4d192231c43978357b681882fc5d162943ada3a0a0d7f age=2h33m10s dirty="0.00 B"`

#### Steps to reproduce the behaviour
Stop bsc 1.0.6
Upgrade to bsc 1.0.7
Start bsc 1.0.7 with the existing datadir 

(i've seen similar behaviour when having copied the data of 1.0.6 to a different machine and starting it there under 1.0.6, but i had assumed this might have been because of ssd rather than nvme and another node running in parallel)

**Comment by terianil** [comment]
I have the same behavior.

**System information**
Geth version: 1.0.7
 OS & Version: Amazon Linux 2
Machine: AWS EC2 c5.2xlarge - 8 CPU, 16GB, EBS 16k IOPS

**Actual behaviour**
BSC sometimes stays out of sync for few hours.
No errors/warns in bsc.log. Just new blocks appears slowly.
While average metrics are still ok: CPU ~35%, MEM 45%, IOPS < 10%
`net.peerCount = 66`

**Comment by koen84** [comment]
It would also be interesting to know what triggers these processes ?
And -if they can be disabled- what the potential consequences are ?

**Comment by kutysam** [comment]
I'm having this also.
Based on my understanding, this is for geth's snapsync which was introduced some versions 1.10.
However, in 1.10, they have a flag of snapshot false which disables this snapshot sync.

I can't seem to find an option in BSC 1.0.7 to disable it though.

**Comment by koen84** [comment]
I'm back to a 2h lag, while that snapshot aborting / resuming is running.  This is really not okay.

**Comment by guagualvcha** [comment]
The snapshots of the current implement usually break when the client been force killed.  I am aware of this and will add an option to close the snapshot.

**Comment by BinanceG11** [comment]
We have responded to the question and will proceed to close the case as we didn't get any additional question after 3days.  Please proceed to join our Discord channel for more discussion at https://discord.com/invite/YPDsUqcwR8

**Comment by koen84** [comment]
@BinanceG11 soooo "won't fix" one of the critical performance impacts ?
Sure, there's a workaround to disable the snapshotting, but for those with full nodes, it'll mean they can't prune, unless they catchup in that snapshotting process, which will take a long time if it's been off.
